build.fsx

```diff
@@ -132,9 +137,59 @@ Target "RunTests" <| fun _ ->
     let xunitToolPath = findToolInSubPath "xunit.console.exe" "src/packages/xunit.runner.console*/tools"
     printfn "Using XUnit runner: %s" xunitToolPath
     xUnit2
-        (fun p -> { p with OutputDir = testOutput; ToolPath = xunitToolPath })
+        (fun p -> { p with HtmlOutputPath = Some(testOutput @@ "xunit.html") })
         xunitTestAssemblies
 
+Target "StartDbContainer" <| fun _ ->
+    let dockerImage = getBuildParamOrDefault "dockerImage" @"microsoft/mssql-server-windows-express"
+    logfn "Starting SQL Express Docker container using image: %s" dockerImage
+    let posh = PowerShell.Create().AddScript(sprintf @"./docker_sql_express.ps1 -dockerImage %s" dockerImage)
+    posh.Invoke() |> Seq.iter (logfn "%O")
+
+    match posh.HadErrors with
+    | true -> posh.Streams.Error |> Seq.iter (logfn "\t %O")
+              failwith "SQL Express Docker container startup encountered an error... failing build"
+    | false -> ()
+
+    match environVarOrNone "container_ip" with
+    | Some x -> logfn "SQL Express Docker container created with IP address: %s" x
+    | None -> failwith "SQL Express Docker container env:container_ip not set... failing build"
+
+Target "PrepAppConfig" <| fun _ -> 
+    let ip = environVarOrNone "container_ip"
+    match ip with
+    | Some ip ->
+        let appConfig = "src/Akka.Persistence.SqlServer.Tests/App.config"
```

Comment:
Yes I agree, I entertained the option of using the `!!` path searching, like what is used in the `RunTests` target for finding assemblies with `*.Tests.dll`.  Could be done if there's the possibility of finding multiple `.Tests.dll` assemblies in a project.